courses/free-courses-ar.md

```diff
@@ -2,10 +2,13 @@
 
 * [Algorithms & Data Structures](#algorithms--data-structures)
 * [Assembly](#assembly)
+  * [Computer Architecture](#computer-architecture)
```

Comment:
This better as 3th level (as same as it was at first commits), like in other listings